<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQModelWidget.ui,v $ </comment>
<comment>   $Revision: 1.3 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2012/05/09 21:32:17 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2012 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQModelWidget</class>
 <widget class="CopasiWidget" name="CQModelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="mpLblTimeUnit">
     <property name="text">
      <string>Time Unit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="mpComboTimeUnit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="mpLblInitialTime">
     <property name="text">
      <string>Initial Time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="mpEditInitialTime">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="Line" name="mpLine1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="mpLblCurrentTime">
     <property name="text">
      <string>Current Time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="mpEditCurrentTime">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="mpComboVolumeUnit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="mpLblVolumeUnit">
     <property name="text">
      <string>Volume Unit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="mpComboAreaUnit"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="mpLblAreaUnit">
     <property name="text">
      <string>Area Unit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="mpComboLengthUnit"/>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="mpLblLengthUnit">
     <property name="text">
      <string>Length Unit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mpLblQuantityUnit">
     <property name="text">
      <string>Quantity Unit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="mpComboQuantityUnit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mpLblModelType">
     <property name="text">
      <string>Rate Law Interpretation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="mpComboModelType"/>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="CQNotes" name="mpNotes" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CQNotes</class>
   <extends>QWidget</extends>
   <header>copasi/UI/CQNotes.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpComboTimeUnit</tabstop>
  <tabstop>mpComboVolumeUnit</tabstop>
  <tabstop>mpComboQuantityUnit</tabstop>
  <tabstop>mpComboAreaUnit</tabstop>
  <tabstop>mpComboModelType</tabstop>
  <tabstop>mpComboLengthUnit</tabstop>
  <tabstop>mpEditInitialTime</tabstop>
  <tabstop>mpEditCurrentTime</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
